server/controllers/boardController.js

```diff
@@ -0,0 +1,33 @@
+import Board from '../models/boardModel';
+import { buildResponse } from '../utils/helpers';
+import {} from '../models/issuesModel';
+
+const getBoardDetails = async (req, res) => {
+  const id = parseInt(req.params.id, 10);
```

Comment:
Put this line after if condition

Reply:
Correct me If I am wrong, the request params comes in form of string right? So I need to parse it to the number and check if the parsed id is number or not.

Reply:
Yes, you are right. @praveshtora.

Reply:
```suggestion
try {
  const id = parseInt(req.params.id, 10);
} catch (err) {
   // unable to parse, it must be not a number.
}
```

Reply:
After parsing the string to a number it always gives a number, so in this case, the if statement is always true even if the passed id is not a number. 
So Instead of checking for typeof used isNaN and then parseInt the value

Reply:
I started with isNaN but eslint did not allow me to. Maybe I will have to push with no-verify then?